test/OfflineTest.php

```diff
@@ -212,11 +234,54 @@ public function testAddingAttributes(
      */
     public function testAddingAttributesFails()
     {
-        $ldap_mod_del = $this->getFunctionMock('Zend\\Ldap', 'ldap_mod_add');
-        $ldap_mod_del->expects($this->once())
+        $ldap_mod_add = $this->getFunctionMock('Zend\\Ldap', 'ldap_mod_add');
+        $ldap_mod_add->expects($this->once())
```

Comment:
Please do not change existing tests. Especially when the test has nothing to do with the newly added features.

Reply:
Oh, I see what this was. testAddingAttributesFails() had clearly been copied from testRemovingAttributesFails(), but the $ldap_mod_del local variable name hadn't been updated. Was just trying to tidy up a bit; it wouldn't have a material impact on the test, but I will revert it.